react/global.css

```diff
@@ -0,0 +1,54 @@
+body {
+    width: 100%;
+}
+.top-100 {
+    top: 100%
+}
+
+.top-50 {
+    top: 50%
+}
+
+.minicart-item {
```

Comment:
You should follow the pattern defined by @brunojdo 

vtex-nomedaapp-componente__subcomponente

Like: 

vtex-shelf__arrow-left

Reply:
Done!